agent/bench-scripts/pbench-user-benchmark

```diff
@@ -32,16 +32,20 @@ benchmark="pbench-user-benchmark"
 # Defaults
 config=""
 tool_group=default
+sysinfo=default
 
 # Process options and arguments
-opts=$(getopt -q -o C: --longoptions "config:,tool-group:" -n "getopt.sh" -- "$@");
+opts=$(getopt -q -o C: --longoptions "config:,tool-group:,sysinfo:" -n "getopt.sh" -- "$@");
 if [ $? -ne 0 ]; then
 	printf -- "$*\n"
 	printf "\n"
 	printf "\t${benchmark}: you specified an invalid option\n\n"
 	printf "\tThe following options are available:\n\n"
 	printf -- "\t\t-C str --config=str            name of the test config\n"
 	printf -- "\t\t       --tool-group=str\n"
+	printf -- "\t\t       --sysinfo=str,          str = comma seperated values of sysinfo to be collected\n"
+	printf -- "\t\t                                    available: none, default, all"
+	pbench-collect-sysinfo --options
```

Comment:
Not sure if this will end with a new line character.  Perhaps merge with the previous line, like:
```
    printf -- "\t\t   available: $(pbench-collect-sysinfo --options)\n"
```